lib/features/login/domain/repositories/login_repository.dart

```diff
@@ -0,0 +1,8 @@
+import 'dart:ffi';
+
+import 'package:firebase_auth/firebase_auth.dart';
+
+abstract class LoginRepository {
+  Future<UserCredential> signIn();
+  Future<void> logOut();
+}
```

Comment:
no need of abstract classs if there is no plan to more than 1 class to implement.
/iknow that CLEAN said to do it but also clean could easily be a bazuca for a mosquito

Reply:
Fixed